linter.py

```diff
@@ -311,7 +311,7 @@ def find_errors(self, output):
 
     def _inline_setting_bool(self, s):
         """Get an inline setting as a bool."""
-        setting = self.get_view_settings().get(s)
+        setting = self.settings.get(s)
         return setting and setting not in ('False', 'false', '0')
```

Comment:
Can we just get rid of this method? The extra coercion here for literal `'False'` etc. is not needed in SL4. (The name of the method `inline_settings` also assumes there are actually settings inline in the code we would read but we don't do this with SL4 anymore.

So there are I think 2 occurrences, and e.g. above L48 `if self.settings.get('coverage') else {}` should be just enough. (The explicit `get` is also only needed because these settings are kind of undocumented and don't have a default value in `defaults`.) 

WDYT?